site/src/templates/doc.js

```diff
@@ -54,37 +54,110 @@ const ClassName = (props: any) => {
-  >
-    {internalCodeStylesClassName => (
-      <Playground
-        css={mq({
-          marginLeft: [-32, -30],
-          marginRight: [-32, -30],
-          marginTop: 16,
-          marginBottom: 16,
-          borderRadius: [0, 8],
-          whiteSpace: 'pre-wrap',
-          wordBreak: 'break-word'
-        })}
-        editorClassName={internalCodeStylesClassName}
-        logoUrl={logoUrl}
-        initialCompiledCode={props.compiled}
-        code={props.code}
-      />
-    )}
-  </ClassName>
-))
+const createLiveCode = memoize(logoUrl => props => {
+  const parameters = getParameters({
+    files: {
+      'package.json': {
+        content: {
+          name: 'emotion-example-base',
+          version: '1.0.0',
+          description: '',
+          keywords: [],
+          homepage: 'https://codesandbox.io/s/new',
+          main: 'src/index.js',
+          dependencies: {
+            '@emotion/core': '0.13.1',
+            '@emotion/css': '0.9.8',
+            '@emotion/keyframes': '0.9.1',
+            '@emotion/style': '0.8.0',
+            '@emotion/styled': '0.10.6',
+            '@emotion/styled-base': '0.10.6',
+            emotion: '9.2.12',
+            'emotion-theming': '9.2.9',
+            facepaint: '1.2.1',
+            react: '16.5.2',
+            'react-dom': '16.5.2',
+            'react-emotion': '9.2.12',
+            'react-scripts': '1.1.5',
+            recompose: '0.30.0'
+          },
+          devDependencies: {},
+          scripts: {
+            start: 'react-scripts start',
+            build: 'react-scripts build',
+            test: 'react-scripts test --env=jsdom',
+            eject: 'react-scripts eject'
+          }
+        }
+      },
+      '.prettierrc': {
+        content: {
+          printWidth: 50,
+          tabWidth: 2,
+          useTabs: false,
+          semi: false,
+          singleQuote: true,
+          trailingComma: 'none',
+          bracketSpacing: true,
+          jsxBracketSameLine: false
+        }
+      },
+      'sandbox.config.json': {
+        content: {
+          infiniteLoopProtection: true,
+          hardReloadOnChange: false
+        }
+      },
+      'src/index.js': {
+        content: `
+          import React from 'react'
+          import { render } from 'react-dom'
+          
+          import Example from './Example.js'
+          
+          render(<Example />, document.getElementById('root'))
+        `
+      },
+      'src/Example.js': {
+        content: props.code
+      }
+    }
+  })
+
+  const url = `https://codesandbox.io/api/v1/sandboxes/define?embed=1&parameters=${parameters}&query=module=%2Fsrc%2FExample.js%26view=split`
+
+  return (
+    <ClassName
+      css={mq({
+        maxWidth: constants.breakpoints[1] + 'em',
+        paddingLeft: [32, 30],
+        paddingRight: [32, 30]
+      })}
+    >
+      {internalCodeStylesClassName => (
+        <div className={internalCodeStylesClassName}>
+          <iframe
+            src={url}
```

Comment:
We could probably use the `onLoad` event on the iframe to display placeholder and reveal iframe when loaded.